docker-compose.yml

```diff
@@ -16,10 +16,10 @@ services:
     environment:
       - POSTGRES_HOST=postgres
     volumes:
-      - .:/code/
+      - .:/code/src
     ports:
       - "8000:8000"
-    command: python3.7 ./manage.py runserver 0.0.0.0:8000
+    command: python ./manage.py runserver 0.0.0.0:8000
```

Comment:
It could be better but : 
- Now it's a virtualenv used in docker, so python is linked to current virtualenv python3 installed
- I set 3.6 by default, because terra-opp lib should support at least this oldest python version
- Future upgrade would continue to work, even if we upgrade minimum python to other 3.x version